src/main/java/uk/gov/hmcts/reform/managecase/repository/DefaultDataStoreRepository.java

```diff
@@ -82,4 +110,86 @@ public void removeCaseUserRoles(List<CaseUserRole> caseUserRoles, String organis
         dataStoreApi.removeCaseUserRoles(new CaseUserRolesRequest(caseUsers));
     }
 
+    @Override
+    public CaseResource submitEventForCase(String caseId,
```

Comment:
**I am tempted (but undecided)** to say given that ACA-85 also submits events but uses simple wrappers for DataStoreApiClient in this class: that maybe our decision to keep this start event + submit event logic in the class may have been the wrong call.   

Is importing elements of ACA-85 DS repository changes + unit tests into this PR and moving the rest into service an option or too late?

Reply:
Agreed to leave it here so it can be unit tested in isolation, but rename it to be specific to request NoC event, for example

`submitNoticeOfChangeRequestEvent`

Reply:
:+1 implemented